node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -89,6 +99,47 @@ func NewOsDeviceConnectivityHelperScsiGeneric(executer executer.ExecuterInterfac
 	}
 }
 
+func (r OsDeviceConnectivityHelperScsiGeneric) GetVolumeIdByVolumePath(volumePath string, volumeId string) (string, error) {
+	logger.Infof("GetVolumeIdByVolumePath: Searching matching volume id for volume path: [%s] ", volumePath)
+	volumeUuidLower, volumeNguid := r.GetNguidFromVolumeId(volumeId)
+	wantedVolumIds := []string{volumeUuidLower, volumeNguid}
```

Comment:
I don't think `wanted` is really used in English as you use it here, and IMO it doesn't add much info either.
```suggestion
	volumeUids := []string{volumeUuidLower, volumeNguid}
```

Reply:
done

Reply:
are you sure? I still see "wanted" in the diff

Reply:
sorry, I thought you meant only in this file, now I removed all of the `wanted`

Reply:
I still see `wanted`...

Reply:
>I **wanted** to

nice